python_modules/dagster/dagster/_core/storage/sql.py

```diff
@@ -227,3 +226,11 @@ def db_subquery(query, name: str = "subquery"):
         return query.subquery(name)
 
     return query.alias(name)
+
+
+def db_fetch_mappings(conn, query: SqlAlchemyQuery) -> Sequence[Any]:
+    """Utility class that allows compatibility between SqlAlchemy 1.3.x, 1.4.x, and 2.x."""
+    if db.__version__.startswith("2."):
```

Comment:
Consider moving these compat functions into a separate module?

Same comment on previous diff about not startswith("1." - maybe make that an IS_SQLALCHEMY_ONE constant?